src/api/libcellml/variable.h

```diff
@@ -467,13 +469,15 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     VariablePtr clone() const;
 
 private:
+    bool doEquals(const EntityPtr &other) const override; /**< Virtual implementation method for equals, @private. */
```

Comment:
This description doesn't match other similar descriptions.

Reply:
Sorry, can't see this one. Looks the same as all the other descriptions to me.

Reply:
I meant that the description for `doEquals()` is similar (in style) to that of, for instance, `Variable()` and `pFunc()` while I would have expected it to look similar to that of, say, `clone()`.

Reply:
54b1dc8 made the `doEquals()` documentation match that used for `doAddComponent()` which is a better match rather than `clone()`.